src/Castle.Core/Components.DictionaryAdapter/DictionaryAdapterFactory.cs

```diff
@@ -440,22 +468,21 @@ private static void PreparePropertyMethod(PropertyDescriptor descriptor, ILGener
 						.AddBehaviors(typeBehaviors.OfType<IDictionaryInitializer>    ().Cast<IDictionaryBehavior>());
 #endif
 
-			CollectProperties(type, property =>
+			CollectProperties(type, (property, reflectedType) =>

```

Comment:
Did this break because I made changes to `InterfaceAttributeUtil`? Or something .NET Core related?